charts/k8ssandra/values.yaml

```diff
@@ -73,10 +73,11 @@ cassandra:
   # resources.limits and resources.requests **must** be defined.
   allowMultipleNodesPerWorker: false
 
-  # -- Cluster-level heap configuration
-  heap:
-   size:
-   newGenSize:
+  # -- Cluster-level heap configuration (optional).
```

Comment:
To be clear with my requested changes, unless we can uncomment `heap` and its children and modify the template to tolerate their blankness: this line needs to lose the `--`.